### DISCUSSION TITLE
听说PaddleOCRv2.8出来了，Umi-OCR是不是可以也往前跨一大步了？

### DISCUSSION THREAD

**Discussion by lforlgg** [hiroi-sora/Umi-OCR#597] [General]
https://github.com/PaddlePaddle/PaddleOCR/releases
PaddleOCR v2.8.0隆重引入了 PaddleOCR 算法模型挑战赛的冠军方案，为 OCR 领域树立了新的标杆。
表格识别方面据说有巨大进步？
作者大佬跟进中？🚀

**Comment by hiroi-sora** [comment]
嗯，看来纯OCR任务变化不大，增加了新的端到端模型和表格模型。

我近期抽时间研究下

**Comment by hiroi-sora** [comment]
简单记录下：

- 跑通了 PPOCR-2.8 官方的 cpp demo 。迁移为 PaddleOCR-json 有一定工作量，但难度可控。
- 官方文档里有不少坑，哈哈。
- 依赖库文件大小比2.6多了 70MB ，越来越膨胀了。还没算上新引入的表格识别、排版分析等模型。
- OCR推理速度比 2.6 没有显著变化，不过初始化速度快了很多。
- 准确率也没有显著变化。
- 文档提到 [C++预处理后处理还未支持SLANet](https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/algorithm_table_slanet.md) ，即算法模型挑战赛榜首的新版表格识别模型用不了，目前只能用旧版表格模型。
- 好在，挑战赛榜首的另一个模型-SVTR识别模型，C++可以支持。

**Reply by lforlgg** [reply]
大佬威武！
这样的话，2.8内核升级性价比就失去了。
1.多了 70MB ，越来越膨胀。
2.OCR推理速度没有变化
3.准确率也没有变化。
4.新版表格识别模型用不了，目前只能用旧版表格模型。
核心的这四点，已经失去采用它的理由了。
大佬辛苦了！

**Comment by hiroi-sora** [comment]
经过一些调查，我们的C++引擎组件也许可以较低成本地升级到 PaddleOCR-2.8 核心推理框架，支持部分最新模型（支持冠军方案的识别模型，但不支持表格模型）。

👉 https://github.com/hiroi-sora/PaddleOCR-json/issues/139#issuecomment-2241723633

比起 2.6 的旧内核，新内核在多方面有提升，但主要缺点是体积 +70MB 。最高倍率打包压缩后，发布体积 +10MB 左右。

感觉升级还是利大于弊，近期有时间的话会弄一弄。
